nixos/modules/services/mail/mailman.nix

```diff
@@ -183,7 +205,17 @@ in {
       (requirePostfixHash [ "config" "local_recipient_maps" ] "postfix_lmtp")
```

Comment:
Since the mailman module already touches postfix config, maybe we should set this in there as well rather than erroring if it isn't set?

Reply:
We used to do this, and I changed it for a reason. The user might want to include other things in that list, and the order matters. So we can’t set it automatically.

Reply:
The order can still be influenced by the user using `mkBefore`/`mkAfter`/`mkOrder`, but I agree that's not pretty and shouldn't be required unconditionally.